### PR TITLE
Support npm install

### PR DESCRIPTION
**Pull request by justinforce**
We now have at least one app that relies on `npm install` running before it'll start, so add that in.

### CODE REVIEW

File: commands/update [hunk 1]
```diff
@@ -115,21 +121,33 @@ function do_migrate() {
...
-# TODO Delete this
-function bundle_and_migrate() {
+function do_npm() {
+  local app=$1; shift
+  local bundle_args=()
+  if [ -f package.json ]; then
+    tell "${BLUE}$app${GREEN}: npm install..."
+    npm install
+  else
+    tell "${BLUE}$app${YELLOW}: No pacakge.json. I won't npm install."
```
Comment (rmacklin) [inline commands/update]: Oops: Typo in `package`

**Comment by rmacklin** [comment]
Thanks 👍

File: commands/update (hunk 1 above)
Comment (justinforce) [inline commands/update]: HA! Good catch. Thanks. I was _this close_ to extracting that string to a variable. There are a few things here that need to be DRY-ed.

Comment (justinforce) [inline commands/update]: Also doesn't use/need `bundle_args` local variable.

**Pull request merged**